packages/odf/components/mcg/state.ts

```diff
@@ -1,5 +1,10 @@
 import { K8sResourceKind } from '@odf/shared/types';
 
+export type ReplicationResources = {
+  bucketClass: K8sResourceKind;
+  objectBucketClaim: K8sResourceKind;
```

Comment:
I think we don't have "ObjectBucketClaimKind", so good to define one now...
```suggestion
  bucketClass: BucketClassKind;
  objectBucketClaim: ObjectBucketClaimKind;
```